schemas/Account.json

```diff
@@ -31,6 +31,10 @@
     "is_admin": {
       "description": "admin flag",
       "type": "boolean"
+    },
+    "disabled": {
+      "description": "Admin users may temporarily disable an account by setting this flag",
```

Comment:
It is permanent, until an admin decides to reverse it. Maybe "Admin users may disabled/enable an account with this flag." or something along those lines?